pkg/process/net/check.go

```diff
@@ -8,8 +8,7 @@ import (
 	"io/ioutil"
 	"net/http"
 
```

Comment:
I think that `pkg/process/net/check.go` was put here so that the definition of `func (r *RemoteSysProbeUtil) GetCheck(check string)` is close to the package owning `RemoteSysProbeUtil`.
But as this function is only used in `pkg/collector/corechecks/ebpf/{oom_kill,tcp_queue_length}.go`, I’m wondering if it would make sense to move the definition of `func (r *RemoteSysProbeUtil) GetCheck(check string)` in `pkg/collector/corechecks/ebpf/`.
WDYT ?

Reply:
It wouldn't be able to be a function on `RemoteSysProbeUtil` then. Perhaps that type doesn't belong in `pkg/process/net` anymore since more than the `process-agent` communicates with the system-probe?